doc/h2load.1.rst

```diff
@@ -229,6 +229,16 @@ OPTIONS
 
     Default: ``4K``
 
+.. option:: --log-file=<FILE>
+
+    Write per-request information to a file as tab-separated
+    columns:  start time  as microseconds since epoch;  HTTP
+    status code;  microseconds  until end of response.  More
+    columns may be added later.  Rows are ordered by end-of-
+    response time  when using  one  worker  thread,  but may
+    appear  slightly  out  of order  with  multiple  threads
+    due to buffering.  Status code is -1 for failed streams.
+
```

Comment:
These manual pages will be generated in some point in a batch using --help option, and it is not required in PR.

Reply:
OK, I've removed this commit from the branch.